src/controllers/customercontroller.js

```diff
@@ -76,6 +76,12 @@ class CustomerController extends OrderController
     {
         MoneyController.updateMoney(customer.order.orderCost());
 
+        SatisfactionController.updateSatisfaction(this.customer.timestamp - this.order.timeStamp);
```

Comment:
There is no such thing as `this.customer`. Use `customer.timestamp`, and `GAME.model.hours` (which is the current time).